src/main/java/com/example/PicturesToNumber/nn/NeuralNetwork.java

```diff
@@ -15,16 +20,21 @@
 /**
  * This class is a model of neural network
  */
+@Component
+@PropertySource(name="appProperties", value="application.properties")
```

Comment:
Why do you specify property source here? you can specify it one time in root class